src/pages/index.tsx

```diff
@@ -42,10 +47,19 @@ export async function getServerSideProps() {
     return { ...lane, collections: updatedCollections };
   });
 
+  const imageID = featuredImageID();
+  const apiUri = await getAPIUri("local_image_id", imageID);
+  const dataFromUri = await apiCall(apiUri.apiUri);
+  const featuredItemObject = {
+    imageID: imageID,
+    uuid: apiUri.uuid,
+    title: dataFromUri.mods.titleInfo.title,
+    href: `${process.env.DC_URL}/items/${apiUri.uuid}`,
```

Comment:
A double slash is added in the URL. You don't need the slash before "items". In the Vercel preview I get a link like this: `"https://qa-digitalcollections.nypl.org//items/5722fe60-5b93-0131-8261-58d385a7bbd0"`.

Reply:
I'd like to remove the '/' from the DC_URL env variable so that DC_URL and API_URL have the same format, I think it looks cleaner to remove it from the variable vs. keeping it in. This will require an update in our local environments, vercel, QA and Prod environments.

Reply:
I second this– I shouldn't have had the slash in there in the first place